api/tests/routes/adage_iframe/get_collective_offer_template_test.py

```diff
@@ -338,35 +332,36 @@ def test_one_template_id_with_one_archived_template(self, eac_client, redactor):
 
         url = url_for(self.endpoint, ids=[offer.id, archived_offer.id])
 
-        # 1. fetch redactor
-        # 2. fetch collective offer and related data
-        # 3. fetch the venue
-        # 4. fetch the venue's images
-        with assert_num_queries(4):
+        with assert_num_queries(self.expected_num_queries):
             response = eac_client.get(url)
 
-            assert response.status_code == 200
-            assert len(response.json["collectiveOffers"]) == 1
-
-            assert response.json["collectiveOffers"][0]["id"] != archived_offer.id
+        assert response.status_code == 200
+        assert len(response.json["collectiveOffers"]) == 1
+        assert response.json["collectiveOffers"][0]["id"] != archived_offer.id
 
     def test_one_template_id_with_one_inactive_template(self, eac_client, redactor):
         offer = educational_factories.CollectiveOfferTemplateFactory()
         inactive_offer = educational_factories.CollectiveOfferTemplateFactory(isActive=False)
 
         url = url_for(self.endpoint, ids=[offer.id, inactive_offer.id])
 
-        # 1. fetch redactor
-        # 2. fetch collective offer and related data
-        # 3. fetch the venue
-        # 4. fetch the venue's images
-        with assert_num_queries(4):
+        with assert_num_queries(self.expected_num_queries):
             response = eac_client.get(url)
 
-            assert response.status_code == 200
-            assert len(response.json["collectiveOffers"]) == 1
+        assert response.status_code == 200
+        assert len(response.json["collectiveOffers"]) == 1
+        assert response.json["collectiveOffers"][0]["id"] != inactive_offer.id
```

Comment:
Même remarque que précédemment  + 
```suggestion
        assert response.json["collectiveOffers"][0]["id"] == offer.id
```
me semble + utile

Reply:
celui ci je l'ai juste déplacé ce n'est pas un test que j'ai ajouté

mais je peux faire ce changement au passage oui